src/devices/philips.ts

```diff
@@ -12,6 +12,13 @@ const e = exposes.presets;
 const ea = exposes.access;
 
 export const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ["LCX030"],
+        model: "LCX030",
```

Comment:
`LCX030` is an internal number, can you use the model number written on the product or philips site? Also rename the `.png` picture accrordingly.